cmd/admin-heal-ops.go

```diff
@@ -883,11 +882,6 @@ func (h *healSequence) healMinioSysMeta(objAPI ObjectLayer, metaPrefix string) f
 				return errHealStopSignalled
 			}
 
-			// Skip metacache entries healing
-			if strings.HasPrefix(object, "buckets/.minio.sys/.metacache/") {
```

Comment:
I am not fully sure why you removed this ? Also do you know if these are cleared automatically after some time ?

Reply:
We don't need this anymore @vadmeste